src/parsers/urdf/model.hxx

```diff
@@ -365,7 +365,10 @@ inline void parseTree (::urdf::LinkConstPtr link, Model & model, const SE3 & pla
         {
           model.appendBodyToJoint(parent_joint_id, jointPlacement, Y, link->name); //Modify the parent inertia in the model
         }
-        
+        else
+        {
+          model.addFrame(link->name, parent_joint_id, nextPlacementOffset, BODY);
```

Comment:
For me, it is still a fixed joint. I would also keep info on both: FIXED_JOINT with joint->name as name and BODY with link->name as body.